app/src/main/res/values/strings.xml

```diff
@@ -72,4 +72,5 @@ A difference to existing projects like OpenBCI is that it will not be necessary
     <string name="bit24_pref_key">bit24</string>
     <string name="advanced_mode_pref_key">advancedMode</string>
     <string name="axis_desc_time">Time</string>
+    <string name="app_version">1.0.0</string>
```

Comment:
This is wrong. Check pslab-android source code to see how it's done. Also please once again, don't add to end of resource files.

Reply:
Okay.

Reply:
Done.